Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.23
+
+WORKDIR /app
+
+COPY go.mod go.sum ./
+
+RUN go mod download
+
+COPY . .
+
+COPY wait-for-it.sh /app/wait-for-it.sh
+RUN chmod +x /app/wait-for-it.sh 
+
+
+RUN CGO_ENABLED=0 GOOS=linux go build -o /worker-service ./main.go
+
+CMD ["./wait-for-it.sh", "postgres:5432", "--", "./wait-for-it.sh", "rabbitmq:5672", "--", "/worker-service"]
```

Comment:
To much happening, in one CMD, lets create a starting [script](https://refine.dev/blog/docker-entrypoint/#how-to-use-a-script-as-entrypoint)